statement.go

```diff
@@ -107,11 +107,10 @@ func statement(tmpl string, args []driver.NamedValue) (string, error) {
 		} else {
 			re = regexp.MustCompile(fmt.Sprintf("@p%d%s", arg.Ordinal, `\b`))
 		}
-		escaped, err := EscapeArg(arg)
+		val, err := EscapeArg(arg)
```

Comment:
@mattdeekay I know invoking `EscapeArg` was not introduced in your PR.

So this is merely a "non-blocking" question (so I understand the context/impact of this change), in which scenario we escape ARG? is this for supporting SDK side query parametrization?

is `EscapeArg(arg)` invoked in any scenario other than query parametrization?

Reply:
I checked and saw that `EscapeArg(arg)` is only involved in query parametrization across the Go SDK